orte/mca/ras/slurm/ras_slurm_component.c

```diff
@@ -106,6 +106,14 @@ static int ras_slurm_register(void)
                                             MCA_BASE_VAR_SCOPE_READONLY,
                                             &mca_ras_slurm_component.rolling_alloc);
 
+    mca_ras_slurm_component.use_all = false;
+    (void) mca_base_component_var_register (component, "use_entire_allocation",
+                                            "Use entire allocation (not just job step nodes) for this application",
+                                            MCA_BASE_VAR_TYPE_BOOL, NULL, 0, 0,
+                                            OPAL_INFO_LVL_9,
```

Comment:
My only nit is that I wouldn't make this level 9 -- perhaps 4 or 5?  (I don't fully grok the use case, so I don't know exactly who will be using this variable -- see https://github.com/open-mpi/ompi/wiki/MCAParamLevels)